components/tools/OmeroPy/src/omero/gateway/__init__.py

```diff
@@ -141,6 +141,53 @@ def fileread_gen(fin, fsize, bufsize):
     fin.close()
 
 
+def countAnnotations(annotationlinks=None):
+    """
+    Count the different (unique) annotions from the
+    provided annotation links
+    """
+
+    counts = {
+        "TagAnnotation": 0,
+        "FileAnnotation": 0,
+        "CommentAnnotation": 0,
+        "LongAnnotation": 0,
+        "MapAnnotation": 0,
+        "OtherAnnotation": 0}
+
+    if annotationlinks is None:
+        return counts
+
+    atypes = {
```

Comment:
this is inconvenient, accordion is now configurable, would you like to take it to the account? How about other way around, building dict of types from annotationlinks